src/store/sandbox/actions.ts

```diff
@@ -40,9 +41,11 @@ export const eraseIdentity = createAction('account/identity/erase');
 export const fetchAccount = createAsyncThunk<AccountPayload, string>(
   'account/fetch',
   // @ts-ignore
-  async (principal, { rejectWithValue, dispatch }) => {
+  async (principal, { rejectWithValue, dispatch, getState }) => {
+    // @ts-ignore
```

Comment:
😱

Reply:
I think this ts-ignore is needed because the thunk action isn't typed correctly but I haven't had time to determine that yet. This pr is the first instance in the app of needing to use a state value within an action (the api server).